src/main/java/org/utplsql/cli/LoggerConfiguration.java

```diff
@@ -0,0 +1,64 @@
+package org.utplsql.cli;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.encoder.PatternLayoutEncoder;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.ConsoleAppender;
+import com.zaxxer.hikari.HikariDataSource;
+import org.slf4j.LoggerFactory;
+import org.utplsql.api.TestRunner;
+
+class LoggerConfiguration {
+
+    static void configure(boolean silent, boolean debug) {
+        if ( silent )
+            configureSilent();
+        else if ( debug )
+            configureDebug();
+        else
+            configureDefault();
+    }
+
+    private static void configureSilent() {
+        Logger root = (Logger)LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME);
+        root.setLevel(Level.OFF);
+    }
+
+    private static void configureDefault() {
```

Comment:
This kind of manual logging configuration seems a bit strange for me...
Isn't default simply equals to INFO logging level, Debug for debug and silent for non?

Reply:
No, because I don't want the default pattern-layout mess to appear on normal informational output. And yes, it's inconvenient that I have to touch so many things, but that's the only way I found to achieve what I want.